bin/templates/project/build.gradle

```diff
@@ -22,7 +22,8 @@ buildscript {
     ext.kotlin_version = '1.3.50'
     repositories {
         google()
-        jcenter()
+        // org.jetbrains.trove4j:trove4j:20160824.
+        maven { url 'https://repo.appspace.com/list/plugins-release/' }
```

Comment:
Can you explain this a little?

From a quick search it looks to me like trove4j is under LGPL, which we would rather avoid supporting in the core.

I think it would be ideal if this could be specified from a plugin rather than from cordova-android itself.

Reply:
Yeah.. Actually, trove4j is used from com.android.tools.build.
```
> Could not find org.jetbrains.trove4j:trove4j:20160824.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/org/jetbrains/trove4j/trove4j/20160824/trove4j-20160824.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project : > com.android.tools.build:gradle:4.0.1 > com.android.tools.build:builder:4.0.1 > com.android.tools:sdk-common:27.0.1
   
```

https://android.googlesource.com/platform/tools/base/+/refs/heads/mirror-goog-studio-master-dev/sdk-common/build.gradle#34

And, it looks like trove4j has not been migrated to mavenCentral or other popular repository yet.

https://youtrack.jetbrains.com/issue/IDEA-261387

So, please let me know if we should wait until Jetbrain resolves it or if there is any concern to use the appspace maven repo.

Reply:
Aargh. So it looks like this is needed by the Android SDK, nothing to do with Cordova. I would favor making the comment more descriptive, like this for example:

    // This Maven URL is needed to find trove4a artifact for Android SDK ref:
    // https://youtrack.jetbrains.com/issue/IDEA-261387

Or we could just wait for them to resolve the issue. It looks like JCenter will not be gone until February 2022 (next year):

https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/

Reply:
Yes, I would prefer to wait. I don't like the idea of adding a whole repository just for one package, which could be difficult to remove later.

Reply:
I agree with you. @breautek I was feeling that it was a little too early to work on this. Let me close it now and may open it again in the future.